### PR TITLE
change README to tell about coverage only, not full project config

### PR DESCRIPTION
**Pull request by geoffray-adde-sonarsource**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -8,25 +8,19 @@
 image:{img-build-status}[Build Status, link={uri-build-status}]
 image:{img-qg-status}[Quality Gate Status,link={uri-qg-status}]
 
-*This project is analysed on https://sonarcloud.io/dashboard?id=sonarsource-cfamily-examples_linux-cmake-gcovr-gh-actions-sc[SonarCloud]!*
+* The test coverage shown here works on both SonarQube and SonarCloud. This project is analysed on https://sonarcloud.io/dashboard?id=sonarsource-cfamily-examples_linux-cmake-gcovr-gh-actions-sc[SonarCloud]!*
```
Comment (arseniy-sonar) [inline README.adoc]:
```suggestion
*The test coverage shown here works on both SonarQube and SonarCloud. This project is analyzed on https://sonarcloud.io/dashboard?id=sonarsource-cfamily-examples_linux-cmake-gcovr-gh-actions-sc[SonarCloud]!*
```

File: README.adoc [hunk 2]
```diff
@@ -38,17 +32,6 @@ You can take a look at the link:sonar-project.properties[sonar-project.propertie
 - https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#parallel-code-scan[Configuring multithreaded execution]
 - https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-steps-using-compilation-database[Using compilation database (compile_commands.json)]
 
-= Linux\CMake
```
Comment (arseniy-sonar) [inline README.adoc:41]: Why remove this section?

File: README.adoc [hunk 3]
```diff
@@ -8,25 +8,19 @@
...
-* Point the `sonar.coverageReportPaths` property to the generated coverage-report file
-. Make sure that you have your token stored as a secret in your repository (`SONAR_TOKEN`). You can request new tokens using https://sonarcloud.io/account/security/[Account/Security page].
+NOTE: This simple example aims at demonstrating code coverage in action. It explains how to add coverage on a project for which the analysis is already configured. Analysis https://github.com/orgs/sonarsource-cfamily-examples/repositories?language=&page=1&q=topic%3Aconfiguration&sort=&type=all[configuration examples] are available here. +
+Adding code coverage is done the same way regardless of whether you use the Build Wrapper or a Compilation Database to configure the analysis.
+
+To get coverage and feed it to the sonar-scanner, use the following procedure.
+
+. Add `--coverage` compilation flag to instrument the binaries
+. Run the test binaries to count covered lines
+. Run `gcovr --sonarqube` to produce the SonarCloud-compatible XML coverage report
```
Comment (arseniy-sonar) [inline README.adoc]:
```suggestion
. Run `gcovr --sonarqube` to produce the SonarQube- and SonarCloud-compatible XML coverage report
```

**Review by arseniy-sonar — CHANGES_REQUESTED** [review]

File: README.adoc [hunk 4]
```diff
@@ -8,25 +8,19 @@
...
-.. Download the https://docs.sonarcloud.io/advanced-setup/ci-based-analysis/sonarscanner-cli/[Sonar Scanner]
-.. Download the https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-steps-using-build-wrapper[Build Wrapper]
-.. Wrap your compilation with the Build Wrapper
-* add `--coverage` compilation flag to instrument the binaries
-.. Run the test binaries to count covered lines
-.. Run `gcovr --sonarqube` to produce the SonarCloud-compatible XML coverage report
-.. Run `sonar-scanner` as the final step
-* Point the `sonar.coverageReportPaths` property to the generated coverage-report file
-. Make sure that you have your token stored as a secret in your repository (`SONAR_TOKEN`). You can request new tokens using https://sonarcloud.io/account/security/[Account/Security page].
+NOTE: This simple example aims at demonstrating code coverage in action. It explains how to add coverage on a project for which the analysis is already configured. Analysis https://github.com/orgs/sonarsource-cfamily-examples/repositories?language=&page=1&q=topic%3Aconfiguration&sort=&type=all[configuration examples] are available here. +
```
Comment (arseniy-sonar) [inline README.adoc]: I am not sure I understood the purpose of the "configuration" topic, IMO it should include the coverage examples as well. To point users to the examples of analysis configuration you can refer to the bottom of this readme file.

File: README.adoc (hunk 2 above)
Comment (geoffray-adde-sonarsource) [reply README.adoc:41]: as agreed, closing this

**Pull request merged**